joplin/base/migrations/0033_auto_20180927_0500.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.0.8 on 2018-09-27 05:00
+
+from django.db import migrations
+from pprint import pprint
+from wagtail.core.blocks.stream_block import StreamValue
+
+def better_steps(apps, schema_editor):
+    '''
+    We can't import the models directly as they may be a newer
+    version than this migration expects. We use the historical versions.
+    '''
+    ServicePage = apps.get_model('base', 'ServicePage')
+    ServicePageStep = apps.get_model('base', 'ServicePageStep')
+
+    for page in ServicePage.objects.all():
+      pageSteps = []
+      for step in ServicePageStep.objects.all():
+        if step.page_id == page.id:
+          streamStep = {
+            'type': 'basic_step',
+            'value': step.step_description
+          }
```

Comment:
For this migration I'm just taking every step we had in the old model and moving it into the new model as a basic step. We're losing our translations moving to a streamfield here so I'm just using the standard description instead of each of the translated ones.